README.md

```diff
@@ -1 +1,38 @@
-# school-library
\ No newline at end of file
+# school-library
```

Comment:
- [ ] It will be nice to add a short description about your project, as this will help visitors to quickly get an idea of what your project is about.